cmd/operator/main.go

```diff
@@ -68,6 +68,8 @@ func init() {
 	// chaos level will be removed once we have a formal tool to inject failures.
 	flag.IntVar(&chaosLevel, "chaos-level", -1, "DO NOT USE IN PRODUCTION - level of chaos injected into the etcd clusters created by the operator.")
 	flag.BoolVar(&printVersion, "version", false, "Show version and quit")
+	// no-global-cluster-resource will be removed once the default behavior to not create global cluster resources is enforced
+	flag.BoolVar(&constants.NoGlobalClusterResource, "no-global-cluster-resource", true, "This operator will not create global resources(CRD, StorageClass) when set to true. This flag needs to be false before the --pv-provisioner flag can be used.")
```

Comment:
this flag is not descriptive. right now, it is pretty much short-term task driven. it is hard for users who have no context to understand why this flag exists.

we need to make the flag more descriptive, and maybe separate the storageclass and crd to two different flags.

Reply:
Going to change it to two flags to make it clearer.